src/modules/Hero/Hero.tsx

```diff
@@ -1,14 +1,95 @@
 import React from "react";
 import Section from "../components/Section";
 import Flare from "../components/Flare";
-
+import Image from "next/image";
+import barcampHero from "../../../public/images/barcampHero.png";
+import { useState } from "react";
+import { useEffect } from "react";
+import Link from "next/link";
+import Marquee from "react-fast-marquee";
+import { Inter } from "next/font/google";
+const inter = Inter({ subsets: ["latin"] });
 const Hero = () => {
+  const [countdown, setCountdown] = useState<{
+    days: string;
+    hours: string;
+    minutes: string;
+    seconds: string;
+  }>({
+    days: "0",
+    hours: "0",
+    minutes: "0",
+    seconds: "0",
+  });
+
+  useEffect(() => {
+    const pad = (n: number) => (n < 10 ? `0${n}` : n.toString());
+    const targetDate = new Date("2023-11-25T00:00:00Z"); // 25/11/2023
+
+    const intervalId = setInterval(() => {
+      const now = new Date().getTime();
+      const difference = targetDate.getTime() - now;
+
+      const days = Math.floor(difference / (1000 * 60 * 60 * 24));
+      const hours = Math.floor(
+        (difference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60),
+      );
+      const minutes = Math.floor((difference % (1000 * 60 * 60)) / (1000 * 60));
+      const seconds = Math.floor((difference % (1000 * 60)) / 1000);
+
+      setCountdown({
+        days: pad(days),
+        hours: pad(hours),
+        minutes: pad(minutes),
+        seconds: pad(seconds),
+      });
+
+      if (difference < 0) {
+        clearInterval(intervalId);
+        setCountdown({ days: "00", hours: "00", minutes: "00", seconds: "00" });
+      }
+    }, 1000);
+
+    return () => clearInterval(intervalId);
+  }, []);
+
   return (
-    <Section id="hero" flexColCenter mockBorderPlsRemove>
-      <Flare positionX="center" positionY="bottom" />
-      <h1 className="text-desktop-h1">Barcamp Bangkhen 11</h1>
-      <h2 className="text-desktop-h3">Coming soon</h2>
-    </Section>
+    <>
+      <div className={inter.className}>
+        <Section id="hero">
+          <Marquee
+            autoFill={true}
+            className=" bg-yellow-500 text-[30px] font-bold text-black md:text-[40px] "
+          >
+            ! FIRST 200 ATTENDEES GET FREE T-SHIRT
+          </Marquee>
+          <div className="mt-8 flex  flex-row flex-wrap  items-center justify-center">
+            <Image
+              className="h-[382px] w-[382px] md:h-[418px] md:w-[433px] "
+              src={barcampHero}
```

Comment:
this way of import image make image flashing on html load

pls use this static image way
![image](https://github.com/barcamp-bangkhen-11/bcbk11-frontend/assets/70707895/32563b1b-4e9a-4168-ba82-f81b8074731f)

Reply:
Got it sub